package.json

```diff
@@ -4,24 +4,14 @@
   "description": "Polyfill of the PointerEvents W3C spec",
   "directories": {
     "doc": "docs",
-    "test": "test"
+    "test": "tests"
   },
   "devDependencies": {
-    "chai": "^1.9.1",
     "chai-spies": "^0.5.1",
     "esperanto": "^0.6.6",
     "grunt": "~0.4.1",
     "grunt-contrib-uglify": "^0.4.0",
-    "grunt-git-authors": "^3.0.0",
-    "grunt-karma": "*",
-    "karma": "~0.12.0",
-    "karma-crbot-reporter": "*",
-    "karma-firefox-launcher": "^0.1.3",
-    "karma-ie-launcher": "*",
-    "karma-mocha": "*",
-    "karma-safari-launcher": "~0.1.1",
-    "karma-script-launcher": "*",
-    "mocha": "*"
+    "grunt-git-authors": "^3.0.0"
```

Comment:
Yes. It totally was there the whole time and I didn’t force push a replacement commit to the branch to make it show up on line 12. Ahem. :blush: